modules/apps/web-experience/site/site-item-selector-api/src/main/java/com/liferay/site/item/selector/criteria/SiteItemSelectorReturnType.java

```diff
@@ -17,6 +17,19 @@
 import com.liferay.item.selector.ItemSelectorReturnType;
 
 /**
+ * This return type should return the following information of a group as a
+ * JSON object:
+ *
+ * <code>
+ * {
+ * 		groupDescriptiveName: The name of the selected group
+ * 		groupId: The group ID of the selected group
+ * 		groupType: The type of the selected group
+ * 		url: The URL of the selected group
+ * 		uuid: The UUID of the selected group
+ * }
```

Comment:
Could you talk with @jhinkey @mwilliams2014?, because they reviewed the javadoc for the other ReturnTypes. The only important thing is it should be clear that the return type is a JSON.